CONTRIBUTING.md

```diff
@@ -179,6 +181,14 @@ If your tests fail, you may wish to look at the logs:
 less _trial_temp/test.log
 ```
 
+To see always see logger output from the app while the tests run,
+set `SYNAPSE_TEST_LOG_LEVEL` and the resulting logs will be in `_trial_temp/test.log`:
```

Comment:
This sentence doesn't make any sense. (Also, we've just mentioned `_trial_temp` - do we really need it again? Leave it in if you prefer, though).

```suggestion
To increase the log level for the tests,  set `SYNAPSE_TEST_LOG_LEVEL`:
```